{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/forms.py

```
@@ -17,7 +17,7 @@ class Meta(admin_forms.UserChangeForm.Meta):  # type: ignore[name-defined]
         {%- endif %}
 
 
-class UserAdminCreationForm(admin_forms.UserCreationForm):
+class UserAdminCreationForm(admin_forms.AdminUserCreationForm):  # type: ignore[name-defined]
```

Ignoring errors is easy but they never drop from a clear sky. 

AdminUserCreationForm is added in https://docs.djangoproject.com/en/5.1/releases/5.1.1/ 

So if Name `AdminUserCreationForm` does not exists I would expect that a version prior to 5.1 is installed?

---

Yes.
So how to do ?

---

I can suggest something like this in the import : ```try:
    from django.contrib.auth.forms import AdminUserCreationForm as AdminUserCreationForm
except (ImportError, ModuleNotFoundError):
    from django.contrib.auth.forms import UserCreationForm as AdminUserCreationForm```

---

`django-stubs` is missing the class, thats why the error is thrown: https://github.com/typeddjango/django-stubs/pull/2555 

Can you add a comment why you are ignoring so we can remove it in the near future?

---

So you ask me to only add a comment into this class ? 
Can you suggest a comment please ? 
Something like : "Ignore because this is available after Django 5.1.1" ?